git-link.el

```diff
@@ -304,6 +304,15 @@ we can prevent that behaviour."
   :type 'list
   :group 'git-link)
 
+;; https://support.atlassian.com/bitbucket-cloud/docs/readme-content/#Extensions-and-Languages
+(defcustom git-link-extensions-rendered-via-annotate-bitbucket '("org" "md" "mkd" "mkdn" "mdown"
```

Comment:
Sounds a wee bit better if we swap `bitbucket` and `annotate`: `git-link-extensions-rendered-via-bitbucket-annotate`